README.md

```diff
@@ -7,8 +7,14 @@
 
 ## Introduction
 
-The iframe module provides an IFrame page type that allows you to embed an IFrame into a page without resorting to
-custom code.
+The IFrame module provides an IFrame page type in the CMS which allows you to embed an IFrame into a page without
+resorting to custom code in your templates or TinyMCE instance.
+ 
+This can help if you have `iframe` disabled in TinyMCE's [valid_elements](https://www.tiny.cloud/docs-3x/reference/configuration/Configuration3x@valid_elements/)
+and do not want to to re-enable it in for a single/specific use-case.
+
+Various attributes of the IFrame can be controlled from CMS fields, such as size and content that surrounds the
+IFrame itself.
```

Comment:
Do you have an example on what attributes are expected when ensuring compatibility with a WAF? I didn't think that WAFs would be affected - my understanding is that the server can't differentiate between a request from an iframe or not - the browser usually has the responsibility of preventing the user from iframe security threats.

Reply:
This was my limited understanding.
Check with TSP for a more authoritative answer I guess.

Reply:
When I get a chance to log into the VPN I'll test this manually to confirm

Reply:
I spoke to one of the infrastructure team about this - the answer was "yeah it could happen, chances are it won’t".

Everyone's setups are different though, so we may as well mention this as a possibility.

Reply:
For the record, I haven't gotten around to testing this on Incapsula